local-cli/link/pods/unregisterNativeModule.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const fs = require('fs');
+const removePodEntry = require('./removePodEntry');
+
+/**
+ * Unregister native module IOS with CocoaPods
+ */
+module.exports = function unregisterNativeModule(dependencyConfig, iOSProject) {
+
+  const podContent = fs.readFileSync(iOSProject.podfile, 'utf8');
+  const removed = removePodEntry(podContent, dependencyConfig.podspec);
```

Comment:
After removing, shall we also call `pod install` to sync the project? What happens right now after you unlink?

Reply:
I don't call `pod install` anywhere. It is up to user to call it later on (it could be also run automatically in build process/Xcode). Right now link/unlink processes only modify configuration files (and with this change only modify `Podfile` file).